File: AtlasWeb/Core/Modules/Tibbi_Surec/Cocuk_Takip_Modulu/ChildGrowthStandardsFormViewModel.cs
//$51446EEF
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;

namespace Core.Controllers
{
    public partial class ChildGrowthStandardsServiceController
    {
        partial void PreScript_ChildGrowthStandardsForm(ChildGrowthStandardsFormViewModel viewModel, ChildGrowthStandards childGrowthStandards, TTObjectContext objectContext)
        {
            viewModel._PatientID = childGrowthStandards.PhysicianApplication.Episode.Patient.ObjectID;
           

        }

        partial void PostScript_ChildGrowthStandardsForm(ChildGrowthStandardsFormViewModel viewModel, ChildGrowthStandards childGrowthStandards, TTDefinitionManagement.TTObjectStateTransitionDef transDef, TTObjectContext objectContext)
        {
       
        }

    }
}

namespace Core.Models
{
    public partial class ChildGrowthStandardsFormViewModel: ISpecialityBasedObjectViewModel
    {
        public Guid _PatientID { get; set; }
        public void AddSpecialityBasedObjectViewModelToContext(TTObjectContext objectContext)
        {
           
        }
    }
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Cocuk_Takip_Modulu/ChildGrowthStandardsFormViewModel.ts
//$7846D474
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { ChildGrowthStandards } from 'NebulaClient/Model/AtlasClientModel';
import { Type } from 'NebulaClient/ClassTransformer';
import { Guid } from 'NebulaClient/Mscorlib/Guid';
export class ChildGrowthStandardsFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.ChildGrowthStandardsFormViewModel, Core';
    @Type(() => ChildGrowthStandards)
    public _ChildGrowthStandards: ChildGrowthStandards = new ChildGrowthStandards();
    @Type(() => Guid)
    public _PatientID: Guid;
}
